src/Microsoft.DotNet.Build.Tasks/PackageFiles/PerfTesting.targets

```diff
@@ -3,17 +3,13 @@
 
   <!-- Perf Runner NuGet package paths -->
   <PropertyGroup Condition="'$(TargetOS)'=='Linux'">
-    <PerfDotNetCliDir>$(TestPath)../perf.dotnetcli</PerfDotNetCliDir>
+    <PerfDotNetCliDir>%24PACKAGE_DIR/perf.dotnetcli</PerfDotNetCliDir>
```

Comment:
MSBuild won't allow me to add a $PACKAGE_DIR here, it forcibly evaluates the PACKAGE_DIR param which is obviously empty

---
In reply to: [69825316](https://github.com/dotnet/buildtools/pull/860#discussion_r69825316) [](ancestors = 69825316)